index.html

```diff
@@ -1,79 +1,226 @@
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
+          </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="card_works_tags">
+              <span>html</span>
+              <span>css</span>
+              <span>javascript</span>
+            </div>
+            <button class="card_works_button">See Project</button>
+         </div>
+         <div class="card_works">
+           <img class="card_works_image" src="./assets/multi_post_2.svg" alt="image">
+           <h2 class="card_works_heading">Multi-Post Stories</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
+          </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="card_works_tags">
+              <span>html</span>
+              <span>css</span>
+              <span>javascript</span>
+            </div>
+            <button class="card_works_button">See Project</button>
+         </div>
+        </section>
+
+        <div class="hero">
+          <section class="about flex">
+          <h1 class="about__title">
+            About <span>Myself</span>
+         </h1>
+         <p class="about__dsc">
+            Hello I'm a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact
+            me.
+         </p>
+         <a href="#" class="head__link">LET'S CONNECT</a>
+         <div class="assets__icons">
+            <ul class="social-media-icons">
+               <li class="li">
+                  <a href="https://twitter.com/DhaPhyn" class="assets link"
+                     ><img src="./assets/twitter.svg.svg" alt="twitter icon"
+                     /></a>
+               </li>
+               <li class="li">
+                  <a href="https://www.linkedin.com/in/daphine-atim-27861422a/"
+                     class="assets link"
+                     ><img src="./assets/linkedin.svg.svg" alt="linkedin icon"
+                     /></a>
+               </li>
+               <li class="li">
+                  <a href="https://medium.com/@hopedaphine19" class="assets link"
+                     ><img src="./assets/medium.svg.svg" alt="medium icon"
+                     /></a>
+               </li>
+               <li class="li">
+                  <a href="https://github.com/Daphineatim" class="assets link"
+                     ><img src="./assets/github.svg.svg" alt="github icon"
+                     /></a>
+                     
```

Comment:
Please ensure that you stick to the design as much as possible (e.g., font, colors, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/154817589-06ce33e7-3d9c-4a44-84da-3a1b47a3ff64.png) | ![image](https://user-images.githubusercontent.com/10332499/154817604-7ae934de-9734-4e8a-b9b7-f3bbb91610a5.png)